src/electron/views/device-connect-view/components/android-setup/device-description.scss

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../../../common/styles/fonts.scss';
+
+.content {
+    display: flex;
+    line-height: 19px;
+    margin-top: 24px;
```

Comment:
I think the vertical alignment between the icon and the text is just slightly off from the figma; I think the exact match for the figma would be

```css
align-items: center;
height: 48px;
margin-top: 16px;
margin-bottom: 16px;
```

You might also consider handling the margins in the step's CSS rather than the DeviceDescription's (generally, spacing between sibling elements should be the responsibility of the component that's composing those sibling elements, especially since we know we'll want to reuse this in a different step with different margin requirements), but if you'd rather leave that for the prompt-choose-device UI PR to handle, that's okay too

Reply:
marginTop and marginBottom are now passed in via the DeviceDescriptionProps